diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 86565bf..9e1a77f 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -926,7 +926,7 @@ static inline void set_protect_slice(struct cfs_rq *cfs_rq, struct sched_entity
 #else // CONFIG_SCHED_BORE
 	u64 slice = normalized_sysctl_sched_base_slice;
 	bool run_to_parity = sched_feat(RUN_TO_PARITY);
-#endif CONFIG_SCHED_BORE
+#endif // CONFIG_SCHED_BORE
 	u64 vprot = se->deadline;
 
 	if (run_to_parity)
@@ -8813,7 +8813,7 @@ static void check_preempt_wakeup_fair(struct rq *rq, struct task_struct *p, int
 	if (run_to_parity && do_preempt_short)
 #else // CONFIG_SCHED_BORE
 	if (sched_feat(RUN_TO_PARITY) && do_preempt_short)
-#endif CONFIG_SCHED_BORE
+#endif // CONFIG_SCHED_BORE
 		update_protect_slice(cfs_rq, se);
 
 	return;
